internal/common/metrics.go

```diff
@@ -0,0 +1,129 @@
+package common
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/giantswarm/clustertest/pkg/client"
+	"github.com/giantswarm/clustertest/pkg/logger"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/giantswarm/cluster-test-suites/internal/helper"
+	"github.com/giantswarm/cluster-test-suites/internal/state"
+)
+
+func runMetrics() {
+	Context("default apps", func() {
+		var mcClient *client.Client
+
+		BeforeEach(func() {
+			mcClient = state.GetFramework().MC()
+		})
+
+		// List of metrics that must be present.
+		metrics := []string{
```

Comment:
Might be best to move this into either the `BeforeEach` or the `It` as otherwise the ordering of code execution can be a bit unexpected if tests (in the future) make changes to this slice.

Reply:
done